crates/control/src/behavior/node.rs

```diff
@@ -43,6 +43,8 @@ pub struct CycleContext {
     pub configuration: Parameter<BehaviorConfiguration, "behavior">,
     pub field_dimensions: Parameter<FieldDimensions, "field_dimensions">,
     pub lost_ball_parameters: Parameter<LostBall, "behavior.lost_ball">,
+
+    pub has_ground_contact: Input<bool, "has_ground_contact">,
```

Comment:
Please group this `Input` with the other `Input` field.

Reply:
Done